src/pages/Home/components/Days/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Day } from './Days'
+import {GlobalSvgSelector} from "../../../../assets/icons/global/GlobalSvgSelector";
```

Comment:
Use the prettier om even save
```suggestion
import { GlobalSvgSelector } from "../../../../assets/icons/global/GlobalSvgSelector";
```